code/game/data_huds.dm

```diff
@@ -18,7 +18,7 @@
 /datum/atom_hud/data
 
 /datum/atom_hud/data/human/medical
-	hud_icons = list(STATUS_HUD, HEALTH_HUD)
+	hud_icons = list(STATUS_HUD, HEALTH_HUD, NANITE_HUD)
```

Comment:
despite me putting this here i don't know if it's fixed so i didn't ping the issue